tests/model/writer.js

```diff
@@ -2086,25 +2117,62 @@ describe( 'Writer', () => {
 			expect( marker.managedUsingOperations ).to.be.false;
 		} );
 
-		it( 'should enable changing marker to be not managed using operation', () => {
+		it( 'should not create additional operation when using operation is not specified', () => {
 			const spy = sinon.spy();
 			model.on( 'applyOperation', spy );
 
-			setMarker( 'name', range );
-			const marker = setMarker( 'name', range, { usingOperation: true } );
+			addMarker( 'name', { range, usingOperation: true } );
+
+			sinon.assert.calledOnce( spy );
+
+			updateMarker( 'name', { range } );
```

Comment:
As above. You don't change range nor usingOperation. I do not get what this test is about.